media-video/subliminal/subliminal-9999.ebuild

```diff
@@ -20,32 +20,33 @@ KEYWORDS=""
 IUSE="test"
 
 RDEPEND="
-	>=dev-python/guessit-2.0.1[${PYTHON_USEDEP}]
+	>=dev-python/appdirs-1.3[${PYTHON_USEDEP}]
 	>=dev-python/babelfish-0.5.2[${PYTHON_USEDEP}]
-	>=dev-python/enzyme-0.4.1[${PYTHON_USEDEP}]
 	>=dev-python/beautifulsoup-4.4.0:4[${PYTHON_USEDEP}]
-	>=dev-python/requests-2.0[${PYTHON_USEDEP}]
+	>=dev-python/chardet-2.3.0[${PYTHON_USEDEP}]
 	>=dev-python/click-4.0[${PYTHON_USEDEP}]
 	>=dev-python/dogpile-cache-0.6.0[${PYTHON_USEDEP}]
-	>=dev-python/stevedore-1.0.0[${PYTHON_USEDEP}]
-	>=dev-python/chardet-2.3.0[${PYTHON_USEDEP}]
+	>=dev-python/enzyme-0.4.1[${PYTHON_USEDEP}]
+	>=dev-python/guessit-2.0.1[${PYTHON_USEDEP}]
 	>=dev-python/pysrt-1.0.1[${PYTHON_USEDEP}]
-	>=dev-python/six-1.9.0[${PYTHON_USEDEP}]
-	>=dev-python/appdirs-1.3[${PYTHON_USEDEP}]
-	>=dev-python/rarfile-2.7[compressed,${PYTHON_USEDEP}]
 	>=dev-python/pytz-2012c[${PYTHON_USEDEP}]
+	>=dev-python/rarfile-2.7[compressed,${PYTHON_USEDEP}]
+	>=dev-python/requests-2.0[${PYTHON_USEDEP}]
+	>=dev-python/six-1.9.0[${PYTHON_USEDEP}]
+	>=dev-python/stevedore-1.0.0[${PYTHON_USEDEP}]
+	dev-python/decorator[${PYTHON_USEDEP}]
 	dev-python/setuptools[${PYTHON_USEDEP}]
 	virtual/python-futures[${PYTHON_USEDEP}]
 "
 DEPEND="${RDEPEND}
 	test? (
-		app-arch/unzip
+		$(python_gen_cond_dep 'dev-python/mock[${PYTHON_USEDEP}]' python2_7)
 		>=dev-python/vcrpy-1.6.1[${PYTHON_USEDEP}]
-		dev-python/pytest[${PYTHON_USEDEP}]
+		app-arch/unzip
```

Comment:
Sort by category + package name, discard the operator.

Reply:
Done.